source/nodejs/adaptivecards/src/serialization.ts

```diff
@@ -83,6 +83,11 @@ export class Version {
         return 0;
     }
 
+    isVNextFeature(): boolean
+    {
+        return isVersionLessOrEqual(Versions.latest, this);
+    }
```

Comment:
```suggestion
    get isPreview(): boolean
    {
        return isVersionLessOrEqual(Versions.latest, this);
    }
```

Reply:
To avoid conflict with IsPreviewMode, I rename it to isPreviewFeature